src/styles/grid.scss

```diff
@@ -0,0 +1,19 @@
+.cards {
+  display: grid;
+  grid-template-columns: 200px;
+  gap: 46px 48px; // вертикальний відступ 48px, горизонтальний 46px
+  padding: 50px 40px;
```

Comment:
Ensure that the padding values are consistent with the design specifications. If the design specifies different padding values, adjust them accordingly.